core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -407,6 +407,18 @@ private static void sendConfirmation(
         readyForNextRequest(context);
     }
 
+    private static void sendInsertConfirmation(
+            JsonQueryProcessorState state,
+            CharSequence keepAliveHeader
+    ) throws PeerDisconnectedException, PeerIsSlowToReadException
+    {
+        final HttpConnectionContext context = state.getHttpConnectionContext();
+        final HttpChunkedResponse response = context.getChunkedResponse();
+        header(response, context, keepAliveHeader, 200);
+        response.sendChunk(true);
+        readyForNextRequest(context);
+    }
+
     private static void sendUpdateConfirmation(
```

Comment:
Both INSERT and UPDATE could return `{"dml":"OK"}` response.

Reply:
@puzpuzpuz sendUpdateConfirmation returns **updatedRecords** count in the response as well, do we want that in insert also?

Reply:
Yes, you're right. UPDATE should return something like `{"dml":"OK","updated":123}`

Reply:
@puzpuzpuz do we want this for insert as well ? {"dml":"OK","inserted":123}?

Reply:
That would require more code changes since we currently don't track the number of inserted rows in `InsertOperation`.